### PR TITLE
Translating team.md

### PR DESCRIPTION
**Pull request by E3saR33**
I've translated team.md to Arabic.
> ### Completed now .

### CODE REVIEW

File: content/community/team.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ---
 id: team
-title: Team
```
Comment (E3saR33) [inline content/community/team.md:3]: Page title in Arabic : الفريق

File: content/community/team.md [hunk 2]
```diff
@@ -117,6 +117,7 @@ React was originally created by [Jordan Walke](https://github.com/jordwalke). To
 * [Tim Yung](https://github.com/yungsters)
 * [Yuzhi Zheng](https://github.com/yuzhi)
 
-This list is not exhaustive.
+هذه القائمة ليست شاملة.
 
-We'd like to give special thanks to [Tom Occhino](https://github.com/tomocchino) and [Adam Wolff](https://github.com/wolffiex) for their guidance and support over the years. We'd also like to thank all the volunteers who [translated React into other languages](https://isreacttranslatedyet.com/).
+نود أن نقدم شكر خاص ل [توم أوتشينو](https://github.com/tomocchino) و [آدم وولف](https://github.com/wolffiex) على مساعدتهم و توجيههم على مر السنين.
```
Comment (Aissaoui-Ahmed) [inline content/community/team.md:122]:
```suggestion
نود أن نقدم شكر خاص ل [Tom Occhino](https://github.com/tomocchino) و [Adam Wolff](https://github.com/wolffiex) على مساعدتهم و توجيههم على مر السنين.
```

Comment (E3saR33) [reply content/community/team.md:122]: 👏🏻👍🏻

File: content/community/team.md [hunk 3]
```diff
@@ -1,86 +1,87 @@
...
 
 ### Brian Vaughn {#brian-vaughn}
 
 ![Brian Vaughn](../images/team/bvaughn.jpg)
 
 [@bvaughn on GitHub](https://github.com/bvaughn) &middot; [@brian\_d\_vaughn on Twitter](https://twitter.com/brian_d_vaughn)
 
-Brian studied art in college and did programming on the side to pay for his education. Eventually, he realized that he enjoys working on open source. Brian has one [one-person band](https://soundcloud.com/brianvaughn/) and two [two-person](https://soundcloud.com/pilotlessdrone) [bands](https://soundcloud.com/pinwurm). He also takes care of the cutest cat in the world.
+Brian قد درس الفن في الكلية و قام بالبرمجة بجانب دراسته ليدفع تكاليف تعليمه. في النهاية, أدرك أنه يستمتع بالعمل في مشاريع مفتوحة المصدر.
+هو أيضاً يعتني بألطف قطة في العالم
```
Comment (Aissaoui-Ahmed) [inline content/community/team.md:32]: ??

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

**Pull request closed without merging**